### DISCUSSION TITLE
Why is the strict in tsconfig.json in /examples/basic-typescript set to false?

### DISCUSSION THREAD

**Discussion by TakahiroHimi** [vercel/swr#1592] [Q&A]
Hello!
Thanks to all the people who develop swr!

In `tsconfig.json` in `/examples/basic-typescript`, `strict: false` is set, so there will be no error even if `useSWR` is used incorrectly.

Like this

```ts
const { data } = useSWR(null, fetch) // An error is expected, but no error occurs.
```

If the first argument of `useSWR` is set to `null`, an error will originally occur.
This confused me when I was trying to fix Issue #1576 .
Can I create a PR where I fix it to `strict: true`?

**Comment by huozhi** [comment]
We can definitely set it to `true` and I've filed a PR updated it already.
The example you showed is not expected to error. When key is null, useSWR will stop revalidations.
check https://swr.vercel.app/docs/conditional-fetching
